tensorrt_llm/_torch/auto_deploy/compile/backends/torch_cudagraph.py

```
@@ -60,6 +60,20 @@ def autotune(*args, **kwargs):
 
+def _safe_reset_cuda_graph(graph: torch.cuda.CUDAGraph, context: str) -> None:
+    """Best-effort reset of a partially-captured CUDA graph.
+
+    graph.reset() can itself raise (e.g. because the CUDA generator state has
+    already been torn down or is inconsistent after a failed capture). Swallow
+    those failures so one failing reset cannot terminate the process from
+    ~CUDAGraph() and cannot mask an earlier, more relevant error.
+    """
+    try:
+        graph.reset()
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)